ocs_ci/utility/azure_utils.py

```
@@ -544,6 +545,30 @@ def create_noobaa_managed_identity(
                 f"--scope {scope} --subscription {subscription_id}"
             )
 
+        test_storage_account = config.AUTH.get("AZURE", {}).get(
+            "STORAGE_ACCOUNT_NAME", ""
+        )
+        if test_storage_account:
+            logger.info(
+                f"Looking up resource ID for test storage account '{test_storage_account}'"
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)